ocaml/idl/schematest.ml

```diff
@@ -0,0 +1,39 @@
+let hash_to_version = [
+    (* (hash, (major, minor)) *)
+    (219580674, (5, 700))
+  ]
+
+let schema_hash : int =
+  [
+    Datamodel_certificate.t
+  ; Datamodel_cluster_host.t
+  ; Datamodel_cluster.t
+  ; Datamodel_diagnostics.t
+  ; Datamodel_host.t
+  ; Datamodel_pool.t
+  ; Datamodel_vm.t
```

Comment:
Is there a way to get this list from the datamodel itself?
E.g. adding a completely new class should require a bump as well. There is a Datamodel.all_system list, would that work?